core/src/main/kotlin/materialui/components/popover/PopoverElementBuilder.kt

```diff
@@ -47,13 +48,38 @@ open class PopoverElementBuilder<Props: PopoverProps>(
     val Tag.transitionDuration: Any? by materialProps
     var Tag.TransitionProps: RProps? by materialProps
 
-    fun Tag.action(actions: (PopoverActions) -> Unit) { action = actions }
```

Comment:
Please do not include modifications that do not affect the behavior as much as possible 🙏